fileWithPRComments.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
```

Comment:
Codacy found an issue: [Move the invocation into the parens that contain the function.](https://dev.k8s.dev.codacy.org/gh/DevK8scodacytestuser/CheckPrCommentsPublicProjectDev/pullRequest?prid=889)